package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Ready-to-go Preact starter project powered by webpack.",
   "scripts": {
     "dev": "cross-env NODE_ENV=development webpack-dev-server --inline --hot --progress",
-    "start": "serve build -s -c 1 && node ./tools/start.cli.js",
+    "start": "serve . -s -c 1 && node ./tools/start.cli.js",
```

Comment:
I wonder whether this will still work.. I can clone in few hours and validate, my gutt feels it shouldn't tho!